PCSX2/pcsx2
# GUI/Qt: Minor text corrections, nitpicks

### Description of Changes
 - Adding the actual corrections from https://github.com/PCSX2/pcsx2/pull/8048 that were not added by https://github.com/PCSX2/pcsx2/pull/8119 .
 - Fixing typos and missing ending periods.
 - Unifying the writing of certain terms:
   - Memory Card uppercased following Sony's writing (taken from PS2 manual).
   - gamefixes/game fixes -> game fixes
   - fast forward/fastforward/fast-forward -> fast-forward (taken from Oxford)
   - slowmotion/slow motion/slow-motion -> slow-motion (following the same convention as before, as I could not find this in Oxford's)
   - framebuffer/frame buffer -> frame buffer (Oxford)
   - Xbox name properly uppercased (for Controller settings)
 - Correcting RA's Rich Presence to separate it from Discord's Rich Presence (after discussions in the Translations channel).
 - Unification of option names in the option area and the hint area.
 - Adding a fix for the (currently broken) Stretch Height/Vertical Stretch tooltip.

### Rationale behind Changes
Making the UI's texts a bit more clean, tidy and uniform. This is by no means a full English editing/proofreading/revision, as that requires a native speaker and I am not.

### Suggested Testing Steps
 - Check the UI overall.
 - Check the Stretch Height/Vertical Stretch tooltip in QT and see if the < > signs are no longer breaking the tooltip.
 - Make sure I did not go Wreck-It Ralph again.
